src/main/java/org/mule/extension/ftp/internal/command/FtpCommand.java

```diff
@@ -111,7 +114,11 @@ protected FtpFileAttributes getFileFromAbsoluteUri(URI uri, boolean requireExist
     }
 
     if (ftpFile.isPresent()) {
-      return new FtpFileAttributes(uri, ftpFile.get());
+      FtpFileAttributes attributes = new FtpFileAttributes(uri, ftpFile.get());
+      if (LOGGER.isTraceEnabled()) {
+        LOGGER.trace("Obtained file attributes {}", attributes);
```

Comment:
are you sure that attributes.toString() returns what you want to log?

Reply:
yes, the implementation uses ReflectionToStringBuilder to retrieve all attributes and build a String. That's what we want to log here.